modules/social_features/social_profile/modules/social_profile_privacy/social_profile_privacy.module

```diff
@@ -229,6 +257,21 @@ function _social_profile_privacy_fields_submit($form, FormStateInterface $form_s
   }
 
   \Drupal::service('cache_tags.invalidator')->invalidateTags($tags);
+
+  if (!isset($profile)) {
+    return;
+  }
+  /** @var \Drupal\profile\ProfileStorageInterface $profile_storage */
+  $profile_storage = \Drupal::entityTypeManager()->getStorage('profile');
+  if (!empty($profile_storage)) {
+    $profile = $profile_storage->load($profile);
+    if ($profile instanceof ProfileInterface) {
+      // We need just save the profile. The profile name will be updated by
```

Comment:
`// We need just save the profile.` to `// We just need to save on the profile.`

Reply:
Updated.